website/docs/configuration/logging.md

```diff
@@ -9,15 +9,26 @@ description: Logging
 
 Logs from FerretDB running on Docker can be accessed through the container.
 
-If Docker was launched with the [quick start](../quickstart-guide/docker.md#setup-with-docker-compose),
+If Docker was launched with [our quick local setup with Docker Compose](../quickstart-guide/docker.md#setup-with-docker-compose),
 the following command can be used to fetch the logs.
 
-```shell
-docker logs -name ferretdb-docker-ferretdb-1
```

Comment:
That command wasn't correct – the container name depends on the directory name where docker-compose.yml was stored